### DISCUSSION TITLE
Markdown tables show properly in editor, but not on live page

### DISCUSSION THREAD

**Discussion by argonaut-network** [requarks/wiki#5378] [Error / Bug Report]
When creating a table in the Markdown editor, it displays properly in the preview sidebar, but does not on the live page. Here is my exact example, copied and pasted from my Wiki.js `:latest` instances:

```
|Minute |Hour |Day of Month |Month |Day of Week |
|--- |--- |--- |--- |--- |
|0 |12 |* |* |* |
```
The table just displays exactly like that in plaintext on the live page.

**Comment by omsfah** [comment]
I've got the samme issue in 2.5.285
Sometimes recreating the page fixes the issue (copy content, delete page, create page, paste content), but not concistenty for me atleast.

**Comment by netthier** [comment]
This issue still exists in the latest version.

EDIT: Turns out that I had to enable "MultiMarkdown Table" in the rendering pipeline. Seems like the editor doesn't respect that setting.

**Reply by HayateLaTech** [reply]
Thanks a lot! X3
Activating that extension fixes the issue. (I think for the preview it's just on at all times, which is confusing but hey..)

**Reply by jarmos-san** [reply]
This should be the accepted answer.

cc @argonaut-network

**Comment by sazeygit** [comment]
Exact steps for any other noobs like myself:

_Settings_ > _Rendering_ > _MutliMarkdown Table_ > _Enable_ > _Apply_ 

and then **resave** the page.
